charts/daps-server/templates/deployment.yaml

```diff
@@ -147,29 +144,17 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           volumeMounts:
+          - mountPath: {{ .Values.omejdn.serverKeyFolderPath }}/config
```

Comment:
@adkumar1 all of this config you've mentioned was removed and implemented at one single volumeMount here.